activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -240,7 +241,7 @@ def [](arg)
         when TZInfo::Timezone
           @lazy_zones_map[arg.name] ||= create(arg.name, nil, arg)
         when Numeric, ActiveSupport::Duration
-          arg *= 3600 if arg.abs <= 13
+          arg *= 3600 if arg.abs <= 60
```

Comment:
I figured this was the best way to future-proof against more extreme time offsets: assume that no offset will ever be less than 1 minute.